packages/editor/package.json

```diff
@@ -92,15 +94,19 @@
     "@hot-loader/react-dom": "^16.11.0",
     "@types/apollo-upload-client": "^8.1.3",
     "@types/express": "^4.17.2",
+    "@types/leaflet": "^1.5.17",
     "@types/node": "^12.12.16",
     "@types/react": "^16.9.16",
     "@types/react-dom": "^16.9.4",
+    "@types/react-leaflet": "^2.5.2",
     "@types/webpack": "^4.41.0",
     "@types/webpack-dev-server": "^3.9.0",
     "babel-loader": "^8.0.6",
     "clean-webpack-plugin": "^3.0.0",
     "crlf": "^1.1.1",
+    "css-loader": "^4.2.1",
```

Comment:
I don't think you need the css-loader and the file-loader. We can check this togehter.